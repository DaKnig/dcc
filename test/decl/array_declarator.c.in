int [5] foo;
