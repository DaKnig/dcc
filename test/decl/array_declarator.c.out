tree representation of array_declarator.c.in:
foo is a 
	array of size
|   |   -> (int)5
of:
	i16 ;
